### PR TITLE
Fixed card overlay

### PR DESCRIPTION
**Pull request by sahiba0915**
Hello team, please review my code for card with text-overlay.

Live review: https://text-overlay.netlify.app/component/card/card.html

Files to check: 
1. card.html
2. card.css

### CODE REVIEW

File: component/card/card.css [hunk 1]
```diff
@@ -74,14 +76,12 @@
 .card h2 {
   margin: 0.6rem 0 0.6rem 0;
```
Comment (akash-1318) [inline component/card/card.css:77]: for these types of value you should use pixel.

**Review by akash-1318 — COMMENTED** [review]
Just added one suggestion besides that everything looks good to me.

File: component/card/card.css (hunk 1 above)
Comment (sahiba0915) [reply component/card/card.css:77]: noted will work on this

**Review by chanchal-panpaliya — COMMENTED** [review]
The class name is consistent, and the code structure is also good and readable. Need to remove the extra space.

File: component/card/card.html [hunk 2]
```diff
@@ -174,20 +174,16 @@ <h2>The Nahargarh Fort</h2>
...
-
-            <p class="card-text">
-              Nahargarh Fort stands on the edge of the Aravalli Hills,
-              overlooking the city of Jaipur in the Indian state of Rajasthan.
-            </p>
+                <div class="text">
+                  <h2 class="heading">The Nahargarh Fort</h2>
+                  <small class="small-txt">Jaipur, Rajasthan</small>
+                </div>   
+
```
Comment (chanchal-panpaliya) [inline component/card/card.html:186]: remove extra space .

Comment (sahiba0915) [reply component/card/card.html:186]: ok will do it

File: component/card/card.css (hunk 1 above)
Comment (pariyar07) [inline component/card/card.css:77]: You can just give this as - margin: 0.6rem 0

File: component/card/card.css [hunk 3]
```diff
@@ -74,14 +76,12 @@
...
   padding: 00.6rem;
```
Comment (pariyar07) [inline component/card/card.css:78]: Why are two 0s in there?

**Review by pariyar07 — COMMENTED** [review]
Left few comments. Otherwise looks good to merge. Great cards designs and color choices.

**Pull request merged**